scripts/looker.py

```diff
@@ -19,95 +19,109 @@
-            "url": _get_looker_event_count_explore_url(
-                looker_namespaces, app.app_name, app.app.get("app_channel")
-            ),
-        }
-        if ping.identifier == "events"
-        else {
-            "name": ping.identifier,
-            "url": _get_looker_ping_explore_url(
-                looker_namespaces,
-                app.app_name,
-                ping.identifier,
-                get_bigquery_ping_table_name(app.app["bq_dataset_family"], ping.identifier),
-                app.app.get("app_channel"),
-            ),
-        }
+        _get_looker_ping_explore(
+            looker_namespaces,
+            app.app_name,
+            ping.identifier,
+            get_bigquery_ping_table_name(app.app["bq_dataset_family"], ping.identifier),
+            app.app.get("app_channel"),
+            app_group,
+        ),
     )
 
 
 def get_looker_explore_metadata_for_metric(
-    looker_namespaces, app, metric, ping_name, ping_has_client_id
+    looker_namespaces, app, app_group, metric, ping_name, ping_has_client_id
 ):
     metric_type = metric.definition["type"]
     metric_name_snakecase = stringcase.snakecase(metric.identifier)
     ping_name_snakecase = stringcase.snakecase(ping_name)
 
-    base_looker_explore_name = "event_counts" if ping_name == "events" else ping_name
-    base_looker_explore_link = (
-        _get_looker_event_count_explore_url(
-            looker_namespaces, app.app_name, app.app.get("app_channel")
+    base_looker_explore = (
+        _get_looker_event_explore(
+            looker_namespaces, app.app_name, app.app.get("app_channel"), app_group
         )
-        if ping_name == "events"
-        else _get_looker_ping_explore_url(
+        if metric_type == "event"
+        else _get_looker_ping_explore(
             looker_namespaces,
             app.app_name,
             ping_name,
             get_bigquery_ping_table_name(app.app["bq_dataset_family"], ping_name),
             app.app.get("app_channel"),
+            app_group,
         )
     )
 
     # we deliberately don't show looker information for deprecated applications
-    if not app.app.get("deprecated") and base_looker_explore_link:
+    if not app.app.get("deprecated") and base_looker_explore:
         looker_metric_link = None
         if metric_type == "event":
             (metric_category, metric_name) = metric.identifier.split(".", 1)
-            looker_metric_link = (
-                base_looker_explore_link
-                + f"&f[events.event_name]=%22{metric_name}%22"
-                + f"&f[events.event_category]=%22{metric_category}%22"
-            )
+            if base_looker_explore["name"] == "event_counts":
+                looker_metric_link = (
+                    base_looker_explore["url"]
+                    + f"&f[events.event_name]=%22{metric_name}%22"
+                    + f"&f[events.event_category]=%22{metric_category}%22"
+                )
+            else:
```

Comment:
small nit: if there's no other event explores other than event_counts and funnel_analysis, should we be more specific here?
```suggestion
            elif base_looker_explore["name"] == "funnel_analysis":
```

Reply:
We should actually assert on the `else`, now that I think about it. Will fix.